scipy/fft/_pocketfft/version.md

```diff
@@ -13,4 +13,6 @@ SciPy currently vendors [pypocketfft][repo] at:
 
         New interface
 
+With the changes in `scipychanges.patch`
```

Comment:
@mreineck are you pushing the same changeset? If so then we don't need this right @peterbell10?

Reply:
I think you still need it, because `scipychanges.patch` refers to a specific (by now pretty outdated) commit in the pypocketfft repository.

Reply:
I see @mreineck has updated pypocketfft, I'll redo this.

Reply:
@mreineck, `scipy.fft` is was actually up to date with pypocketfft master as of a few hours ago (#10393 merged).

Reply:
Sorry, I wasn't aware of this. I still assumed that this kind of synchronisation would be done once the dust of the first merge had settled.